yongkeun/week2/과제/style.css

```
@@ -0,0 +1,31 @@
+body {
+    height: 100vh;
+    background-color:coral;
+    display:flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.mainbox {
```

```suggestion
.main-box {
```
html 코드에 리뷰를 남겨서 아시겠지만 단어의 연결은 "-" 사용하기~